front/src/modules/rollingStock/components/RollingStockEditor/RollingStockEditorFormHelpers.tsx

```diff
@@ -145,6 +145,7 @@ const RollingStockEditorParameterFormColumn = ({
           >
             <InputGroupSNCF
               id={property.title}
+              inputDataTestId={`${property.title}-input`}
```

Comment:
camelCase or kebab-case for property names?

Reply:
Here inputDataTestId is props not a property